src/components/Paragraph/README.md

```diff
@@ -1,61 +1,105 @@
-## Available Sizes
-
-### Large Paragraph
+A `Paragraph` is a self-contained unit of one or more coherent sentences related to a single topic or idea.
 
 ```jsx
 <Paragraph size="large">
 	Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut bibendum hendrerit
-	augue gravida hendrerit. Vestibulum ornare eget lacus cursus rhoncus. Cras nec
-	lectus in sem viverra volutpat. Fusce placerat nunc eu libero egestas, sit
-	amet condimentum turpis ornare. Sed hendrerit metus elit, ac sollicitudin enim
-	vulputate et. Ut fermentum venenatis auctor. Mauris augue augue, hendrerit
-	quis risus eget, imperdiet feugiat odio. Nullam porta sollicitudin ipsum eu
-	varius. Nam hendrerit eleifend sagittis. Maecenas in sodales libero, nec
-	ultrices ante. Ut et sem nibh. Vivamus fringilla pharetra justo aliquet
-	pharetra.
+	augue gravida hendrerit.
+</Paragraph>
+
+<Paragraph>
+	Vestibulum ornare eget lacus cursus rhoncus. Cras nec
+	lectus in sem viverra volutpat.
 </Paragraph>
 ```
 
-### Medium Paragraph
+## Usage
+
+Use a `Paragraph` for any body copy comprising of one or more full sentences.
+
+Don't use a `Paragraph` for:
+- A single text link or call to action—use a [Button](../#/Function/Components/Button) instead.
```

Comment:
Can we highlight "call to action" or separate it a bit better? I feel like this is all a bit tough to parse really quickly. I love me some dashes too but separate sentences might do better here. 🤷🏻‍♂️

```suggestion
- A single text link or _call to action_—use a [Button](../#/Function/Components/Button) instead.
```

Reply:
Fun fact: my "component template" that I'm working on actually sets these up in separate sentences, so I think I agree.